django/db/models/fields/related.py

```diff
@@ -572,11 +572,16 @@ def deconstruct(self):
             kwargs['parent_link'] = self.remote_field.parent_link
         # Work out string form of "to"
         if isinstance(self.remote_field.model, str):
-            kwargs['to'] = self.remote_field.model
+            # Force model name to lower case
+            if "." in self.remote_field.model:
+                app_label, _dot, rest = self.remote_field.model.partition(".")
+                kwargs['to'] = "%s.%s" % (app_label, rest.lower())
+            else:
+                kwargs['to'] = self.remote_field.model.lower()
         else:
             kwargs['to'] = "%s.%s" % (
                 self.remote_field.model._meta.app_label,
-                self.remote_field.model._meta.object_name,
+                self.remote_field.model._meta.model_name,
```

Comment:
This could be replaced by `self.remote_field.model._meta.label_lower`

https://github.com/django/django/blob/c1b24718e05ea474955777d7bc4d9d5634560cd5/django/db/models/options.py#L136-L138

Reply:
Done!